style/desktop-style.css

```diff
@@ -0,0 +1,133 @@
+@media (min-width: 768px) {
+  body {
+    max-width: 144rem;
+    min-height: auto;
+    font-size: 1rem;
+  }
+
+  .flex-container {
+    background-image: url(../images/Header.png);
+    background-size: cover;
+    padding: 160px 260px 90px 130px;
+  }
+
+  .my-button {
+    display: none;
+  }
+
+  .header ul {
+    list-style: none;
+    display: flex;
+    justify-content: end;
+    gap: 1rem;
+    width: 100%;
+  }
+
+  .name {
+    display: flex;
+    margin: 0;
+  }
+
+  .card-container {
+    min-width: 100%;
+  }
+
+  .card {
+    display: flex;
+    min-width: 30rem;
+  }
+
+  .cardpic {
+    width: 120%;
+    height: 96%;
+  }
+
+  .card1-description-p {
+    display: none;
+  }
+
+  .card1-description-alt-p {
+    display: flex;
+    font-size: 2.5rem;
+  }
+
+  .card-title {
+    display: none;
+  }
+
+  .card-title-two {
+    display: flex;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 2rem;
+    line-height: 5rem;
+    color: #091e42;
+    padding: top 0;
+    margin: 0;
+  }
+
+  .cardpic2 {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .desktop {
+    padding-left: 2rem;
+    margin: 0;
+  }
+
+  .connect li {
+    transition: transform 0.2s;
+  }
+
+  .language-fream li {
+    width: 122px;
+    height: 120px;
+    margin-right: 10px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .connect li:hover {
+    transform: scale(2);
+  }
+
+  .about-me {
+    display: flex;
+    flex-direction: row;
+    padding-bottom: 3rem;
+  }
+
+  .about-me1 {
+    width: 50%;
+    margin-right: 46px;
+  }
+
+  .language-fream ul {
+    display: flex;
+    flex-direction: row;
+  }
```

Comment:
- [ ] KIndly make sure you used the correct `width` for each block in the **About me section**. From the Figma design, there are 2 blocks, the left one takes 40% and the right one the remaining 60% of the page's width, and the contents of each of them are aligned to the right. Try to fix this and your project will stick to the Figma design.
- [ ] Kindly make sure you use the correct **color** on titles text like **Languages**(and its content), **Skills**, and **Frameworks** to make your project stick to the Figma design.
### Figma design
![ab_fi](https://user-images.githubusercontent.com/83042742/173111392-46aafca6-b857-4870-977a-7b592099e7e0.png)
### Your design
![ab_st](https://user-images.githubusercontent.com/83042742/173111450-1bc50da8-dfa7-4e74-b684-00507d916633.png)